api/src/test/resources/org/openmrs/module/stockmanagement/api/StockItemsImport.csv

```diff
@@ -1,10 +1,10 @@
-2,456,yes,7,6,20,20,1,TEST 2,,National Medical Stores,50,20,13,20
-3,,no,7,6,20,20,10,TEST 3,T3,nationalMedical Stores,200,20,450,20
-11,,1,7,6,22,22,2,TEST 11,T11,National-Medical-Stores,100,22,1000,22,
-12,6,0,7,6,23,23,30,TEST 12,,,500,23,500,23
-,5497,yes,7,6,20,20,1,TEST 2,,Uganda Medical Stores,50,20,13,20
-12,6,0,7,6,24,24,30,TEST 121,,,500,24,500,24
-2,456,yes,7,6,20,20,1,TEST 2,,National Medical Stores,50,20,13,20
-3,,no,7,6,20,20,10,TEST 3,T3,nationalMedical Stores,200,20,450,20
-11,,1,7,6,22,22,2,TEST 11,T11,National-Medical-Stores,100,22,1000,22,
-,5497,0,7,6,20,20,1,TEST 2,,UgandaMedical Stores,50,20,13,20
\ No newline at end of file
+2,456,yes,7,6,20,20,1,TEST 2,,National Medical Stores,50,20,13,20,NMS-001
+3,,no,7,6,20,20,10,TEST 3,T3,nationalMedical Stores,200,20,450,20,NMS-002
+11,,1,7,6,22,22,2,TEST 11,T11,National-Medical-Stores,100,22,1000,22,NMS-004
+12,6,0,7,6,23,23,30,TEST 12,,,500,23,500,23,
+,5497,yes,7,6,20,20,1,TEST 2,,Uganda Medical Stores,50,20,13,20,NMS-005
```

Comment:
Do you think we should adapt such to our Kenyan context?

Reply:
On the stock tem reference functionality, am still trying to wrap my head around it's use case